app/models/miq_server.rb

```diff
@@ -623,8 +623,8 @@ def messaging_health_check
     broker = MiqQueue.messaging_client("health_check")
     return if broker.nil?
 
-    broker.publish_message(:service => "messaging-health-check", :message => "health check", :payload => {})
-    broker.subscribe_messages(:service => "messaging-health-check") { break }
+    broker.publish_message(:service => "manageiq.liveness-check", :message => "test message", :payload => {})
+    broker.subscribe_messages(:service => "manageiq.liveness-check") { break }
```

Comment:
@nasark re: my question about what it will do currently... we're in the `messaging_health_check` method with the logging on line 629 also saying `messaging health check failed` so I'm curious if the naming of this method / logging is wrong or we're using the wrong topic.  This is why I'm asking.  I'm confused.

Reply:
@jrafanie this is for consistency with the 'messaging-ready' script: https://github.com/ManageIQ/manageiq-appliance/blob/master/COPY/usr/bin/manageiq-messaging-ready#L34-L35

We noticed that we were using two different topics for the same concept

Reply:
Ah, I see... thanks!